Makefile

```diff
@@ -53,8 +53,7 @@ install-kubebuilder:
 
 .PHONY: install-kustomize
 install-kustomize:
-	curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s 3.8.6
-	mv kustomize ${KUSTOMIZE_INSTALL_DIR}/
+	@script/install-kustomize.sh
```

Comment:
if it does not take the destination directory, will [this](https://github.com/shipa-corp/ketch/blob/main/.github/workflows/release.yaml#L35) work?

Reply:
Nope, good catch.  I'll fix it.